app/utils/chainUtils.ts

```diff
@@ -60,56 +60,42 @@ export const CHAIN_DOCS_URL: KeyValuePair<string> = {
 }
 
 export const evmChains = [
-  "amoy-testnet-archival",
-  "arbitrum-one",
-  "arbitrum-sepolia-archival",
-  "avax-archival",
-  "avax-dfk",
-  "avax-mainnet",
-  "base-mainnet",
-  "base-testnet",
-  "blast-archival",
-  "boba-mainnet",
-  "bsc-archival",
-  "bsc-mainnet",
-  "celo-mainnet",
-  "eth-archival",
-  "eth-goerli",
-  "eth-mainnet",
-  "eth-trace",
-  "evmos-mainnet",
-  "fraxtal-archival",
-  "fantom-mainnet",
-  "fuse-mainnet",
-  "fuse-archival",
-  "gnosischain-mainnet",
-  "gnosischain-archival",
-  "goerli-archival",
-  "harmony-0",
-  "holesky-fullnode-testnet",
-  "iotex-mainnet",
-  "kava-mainnet",
-  "kava-mainnet-archival",
-  "klaytn-mainnet",
-  "metis-mainnet",
-  "moonbeam-mainnet",
-  "moonriver-mainnet",
-  "oKc-mainnet",
-  "oasys-mainnet",
-  "oasys-mainnet-archival",
-  "opbnb-archival",
-  "optimism-mainnet",
-  "optimism-archival",
-  "optimism-sepolia-archival",
-  "poly-mainnet",
-  "poly-archival",
-  "polygon-mumbai",
-  "polygon-zkevm-mainnet",
-  "scroll-mainnet",
-  "scroll-testnet",
-  "sepolia",
-  "sepolia-archival",
-  "zksync-era-mainnet",
+  "F003", // avax
```

Comment:
Leaving it up to @fredteumer for manual review.

This is what we have onchain right now.

```json
{
  "param_key": "pocketcore/SupportedBlockchains",
  "param_value": "[\"0001\",\"0003\",\"0004\",\"0005\",\"0006\",\"0009\",\"000A\",\"000B\",\"000C\",\"000F\",\"0010\",\"0012\",\"0021\",\"0022\",\"0024\",\"0025\",\"0026\",\"0027\",\"0028\",\"0040\",\"0044\",\"0046\",\"0047\",\"0048\",\"0049\",\"0050\",\"0051\",\"0052\",\"0053\",\"0054\",\"0056\",\"0057\",\"0058\",\"0059\",\"0060\",\"0061\",\"0063\",\"0065\",\"0066\",\"0067\",\"0068\",\"0069\",\"0070\",\"0071\",\"0072\",\"0074\",\"0075\",\"0076\",\"0077\",\"0078\",\"0079\",\"0080\",\"0081\",\"0082\",\"0083\",\"0084\",\"03DF\",\"7A00\",\"7A10\",\"A001\",\"A003\",\"A053\",\"A085\",\"A086\",\"A087\",\"A088\",\"A089\",\"A090\",\"A091\",\"A092\",\"A0CA\",\"A0CB\",\"A0CC\",\"A0CD\",\"B021\",\"B081\",\"BE2A\",\"C006\",\"E021\",\"F000\",\"F001\",\"F002\",\"F003\",\"F004\",\"F005\",\"F006\",\"F007\",\"F008\",\"F009\",\"F00A\",\"F00B\",\"F00C\",\"F00D\",\"F00E\",\"F00F\",\"F010\",\"F011\",\"F012\",\"F013\",\"F014\",\"F015\",\"F016\",\"F017\",\"F018\",\"F019\",\"F01A\",\"F01B\",\"F01C\",\"F01D\",\"F01E\",\"F01F\",\"F020\",\"F021\",\"F022\",\"F023\",\"F024\",\"F025\",\"F026\",\"F027\",\"F028\",\"F029\",\"F02A\",\"F02B\"]"
}
```

Reply:
@commoddity please add:
`"F02A, // zklink`

only one miss, looks good otherwise!

Reply:
> @commoddity please add: `"F02A", // zklink`
> 
> only one miss, looks good otherwise!

Added to the file, PTAL.